docs/source/advanced/backends.rst

```diff
@@ -1,14 +1,108 @@
+    | Devices                                    | Host Core                                     | Host Cores                                                                    | Host Core                                                                    | Host Cores                                                                      | Host Cores                                                                                                                         | NVIDIA GPUs                                      |
+    +--------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | Lib/API                                    | n/a                                           | std::thread                                                                   | boost::fibers::fiber                                                         | OpenMP 2.0                                                                      | OpenMP 4.0                                                                                                                         | CUDA 9.0+                                        |
+    +--------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | Kernel execution                           | n/a                                           | std::thread(kernel)                                                           | boost::fibers::fiber(kernel)                                                 | ompsetdynamic(0), #pragma omp parallel numthreads(iNumKernelsInBlock)           | #pragma omp target, #pragma omp teams numteams(...) threadlimit(...), #pragma omp distribute, #pragma omp parallel numthreads(...) | cudaConfigureCall, cudaSetupArgument, cudaLaunch |
```

Comment:
For the Serial `backend` `Kernel execution` should not be `n/a`, as it can execute kernels. Perhaps the previous line `Lib/API` should be also change accordingly.

Reply:
But what should I write? No special function or library is used. Maybe something like `bare`, `pure C++`, `no library` and `C++ function call`?

Reply:
I guess something like it. Or Lib/API = standard C++, and kernel execution = sequential